Units/parser-c.r/bug1458930.c.d/expected.tags

```diff
@@ -1,2 +1,2 @@
-x	input.c	/^char x();$/;"	p	file:
-y	input.c	/^wchar_t y();$/;"	p	file:
+x	input.c	/^char x();$/;"	p
```

Comment:
Heh... this is a good question.

The direction I've gone for now is:
   "whenever a symbol/declaration may be visible in any way from another file it's not in file scope".

This means that any variable/function that is not declared static is never limited to file scope because it's enough to declare it as extern somewhere else to be able to access it.

However, as you suggest, a function prototype, especially in a C file is controversial...

Don't know... I can change it if you think that the parser should follow the old rules.

As a side note: it's perfectly possible to include a .c file into another. So this whole "file scope" thing is not 100% reliable. In my editor I just enable all tags and when browsing I let the user choose which one he wants to jump to.